packages/output/src/utils/styles.js

```diff
@@ -135,6 +135,7 @@ function CustomStyles() {
               amp-story-grid-layer.align-bottom {
                 align-content: end;
                 padding: 0;
+                max-height: calc(100vh - 148px);
```

Comment:
Let's add a comment explaining where 148px is coming from.

IIRC it's 2x the size of the CTA layer, so perhaps we can even do `calc(100vh - ( 2 * 74px) );` to make that easier to understand.